### DISCUSSION TITLE
Add move sounds

### DISCUSSION THREAD

**Discussion by ashishpapanai** [ashishpapanai/chessJS#97] [Ideas]
**Add move sounds that would play automatically after any piece is moved.** 

- [ ] Sound for moves after the move is made by either side. 

- [ ]  Different sounds for casting, piece capture, checks and checkmate

**Comment by neeagwl** [comment]
I would like to work on this issue under swoc2021.

**Comment by ashishpapanai** [comment]
@neeagwl Sure, Please send the idea/approach in the ChessJS channel or to me on slack.
